aws/README.md

```diff
@@ -0,0 +1,61 @@
+# Deploy to AWS (Beta)
+
+> **Note:** The AWS deployment template is still in **BETA** phase. You may encounter failures during the deployment. In that case, please let us know the issue.
+
+This template will create a fully-working IPSec/L2TP VPN server on AWS (Amazon Web Service). Please make sure to check the [pricing details](https://aws.amazon.com/ec2/pricing/on-demand/) of Virtual Machine on EC2 before starting the launch sequence.
+
+You can also use `t2.micro` instance as your server for your deployment, which is free of charge within the first year since your AWS account is registered. For more information on AWS free usage tier, go to [this page](https://aws.amazon.com/free/).
+
+## Available Customization Parameters:
+
+- AWS EC2 Instance Type
+- OS for your VPN Server (Ubuntu16.04, Ubuntu18.04 or Debian9-stretch)
+> **Note:** To use Debian9 on EC2, to need to [**subscribe**](https://aws.amazon.com/marketplace/pp/B073HW9SP3) first.
+- Your VPN username
+- Your VPN password
+- IPSec PSK (pre-shared key)
+
+> When choosing your username and password, do not enter special characters like `" ' \`.
+
+Make sure to do this with an **AWS ROOT ACCOUNT** or an **IAM ACCOUNT** with **ADMINISTRATION PRVILEGE**. 
+
+Download the template file [**here**](https://raw.githubusercontent.com/scottpedia/setup-ipsec-vpn/master/aws/cloudformation-template-ipsec) and upload it as the template source in the stack creation wizard.
```

Comment:
Can you change the link to point to this repo instead? i.e.
https://raw.githubusercontent.com/hwdsl2/setup-ipsec-vpn/master/aws/cloudformation-template-ipsec
Users will probably need to right-click on the link and select "Save Link As..." in order to save this template, instead of displaying it in the browser. Perhaps change the wording to let users right-click and save.